engine/remote-components/src/main/java/org/datacleaner/components/remote/RemoteTransformer.java

```diff
@@ -75,6 +75,8 @@
     private ComponentRESTClient client;
     private Map<String, Object> configuredProperties = new TreeMap<>();
 
+    private RuntimeException lastException = null;
```

Comment:
I would suggest to use something that is concurrent / thread-safe instead of an assigned exception. So here I would have
```private final AtomicBoolean failed = new AtomicBoolean(false);```